benthos-plugins/processor/embeddings/openai/README.adoc

```diff
@@ -102,48 +101,49 @@ ATLAS_COLLECTION="Embeddings"
 ATLAS_INDEX="vector_index"
 ----
 
-The Python files in the `demo` directory will load these variables directly from the `.env` file, but you also need to set them in the environment for Benthos configuration interpolation:
+The Python files in the `demo` directory will load these variables from the `.env` file, but you also need to set them in the environment for Benthos configuration interpolation:
 
 [source,bash]
 ----
-% export $(grep -v '^#' ./demo/.env | xargs)
+export $(grep -v '^#' ./demo/.env | xargs)
 ----
 
 === Create Python virtual environment
 
-Last but not least, create the Python virtual environment in the `demo` directory:
+Create the Python virtual environment in the `demo` directory:
 
 [source,bash]
 ----
-% cd demo
-% python3 -m venv env
-% source env/bin/activate
-% pip install -r requirements.txt
+cd demo
+python3 -m venv env
+source env/bin/activate
+pip install -r requirements.txt
 ----
 
 == Run the lab
 
-The Lab comprises of three parts:
+This lab has three parts:
 
 . Use *LangChain's* `WebBaseLoader` and `RecursiveCharacterTextSplitter` to generate chunks of text from the BBC Sport website and send each chunk to a Redpanda topic named `documents`.
 . Use *Redpanda Connect* to consume the messages from the `documents` topic and pass each message through a custom processor that calls *OpenAI's embeddings API* to retrieve the vector embeddings for the text. The enriched messages are then inserted into a *MongoDB Atlas* database collection that has a vector search index.
 . Complete the RAG pipeline by using *LangChain* to retrieve similar texts from the *MongoDB Atlas* database and add that context alongside a user question to a prompt that is sent to OpenAI's new `gpt-4o` model.
 
 === Start Redpanda Connect
 
+In a new terminal window, start Redpanda Connect with the custom OpenAI processor:
+
 [source,bash]
 ----
-#
-# Terminal 1. Start Redpanda Connect with custom OpenAI processor.
-#
-% go test
-PASS
-ok  	benthos-embeddings	2.472s
+go test
+go build
+export $(grep -v '^#' ./demo/.env | xargs)
+./benthos-embeddings -c demo/atlas_demo.yaml --log.level debug
```

Comment:
This must be benthos or it's just an example folder?

Reply:
It's the package name - updated it